lib/cisco_node_utils/feature.rb

```diff
@@ -23,6 +23,25 @@ class Feature < NodeUtil
     # however, for test purposes it is sometimes convenient to support
     # feature disablement for cleanup purposes.
     # ---------------------------
+    def self.bfd_enable
+      return if bfd_enabled?
+      config_set('feature', 'bfd', state: '')
+    end
+
+    def self.bfd_disable
+      return unless bfd_enabled?
+      config_set('feature', 'bfd', state: 'no')
```

Comment:
The reasoning in the FP case was that there was no other meaningful way to turn off FP other than explicitly setting each FP property back to its default (which I still advocate). PIM is another example of this but in the PIM case it just defaults its properties (just one at the moment).

My take on this is that if BFD needs a destroy method then it should follow the PIM model and just set all properties to default; in which case the bfd object will still exist after a destroy but it will be more like an interface object after a destroy.